src/utils/WorkspacesUtils.js

```diff
@@ -71,62 +71,65 @@ const checkDatasetManagerConfiguration = (workspace) => {
   const logWarning = (warning) => console.warn(`Dataset manager configuration: ${warning}`);
   const config = workspace?.webApp?.options?.datasetManager;
   if (config == null) return;
-  if (!(config instanceof Object)) {
+  if (!(config instanceof Object) || config instanceof Array) {
     logWarning('"datasetManager" must be an Object');
     return;
   }
 
   const { graphIndicators, categories, queries } = config;
 
-  if (queries != null && !(queries instanceof Array)) {
-    logWarning('property "queries" must be an Array');
-  } else {
-    const allQueriesIds = [];
-    queries.forEach((query, index) => {
-      if (query.id == null) logWarning(`in queries, item #${index} has no id`);
-      else if (allQueriesIds.includes(query.id))
-        logWarning(`in queries, item #${index} uses a query id that already exists (id: ${query.id}).`);
-      else allQueriesIds.push(query.id);
-
-      if (query.query == null) logWarning(`in queries, item #${index} (id: ${query?.id}) has no query statement`);
-    });
+  const isQueriesValid = queries instanceof Array;
+  if (queries != null) {
```

Comment:
In the case queries is _null_ maybe we should put an additional logWarning in a _else_ block.